ironfish-cli/src/commands/chain/broadcast.ts

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CliUx } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class BroadcastCommand extends IronfishCommand {
+  static description = `Broadcast a transaction to the network`
+
+  static flags = {
+    ...RemoteFlags,
+  }
+
+  static args = [
+    {
+      name: 'transaction',
+      required: true,
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
+      description: 'The transaction in hex encoding',
+    },
+  ]
+
+  async start(): Promise<void> {
+    const { args } = await this.parse(BroadcastCommand)
+    const transaction = args.transaction as string
+
+    CliUx.ux.action.start(`Broadcasting transaction`)
+    const client = await this.sdk.connectRpc()
+    const response = await client.broadcastTransaction({ transaction })
+    CliUx.ux.action.stop()
+
+    if (!response.content?.hash) {
+      this.log(`Transaction broadcast failed: ${response.status});
+      }`)
+      return
+    }
```

Comment:
I'm not sure this is reachable: I think that `client.broadcastTransaction` will propagate the error from the RPC

This is a test with an invalid transaction with a zero fee:
```
fish chain:broadcast 010100000000000000020000000000000000000000000000000000000000000000000000000000000073b70000329c3cc62c0cb7cb662a18c22931f05b49af8b91c335ad16dd55c9ad53305c178533b6130acd5011122e5c027c2908203b39a3ecbca16f578787e4d2d13ffc53ad795d56ad923362940521b6e62d5d598f599c0a2130af9d245a9970bcdd53935069de22ffdfc533c7d6d40d84ccbc4fe781fae1c2339f5864020c84bb020be0004170b171a81c190685b8cacf6faf890900383dab657d4a1dfa10629dcbfab245398fb9b453969179b8ce996f8232d187d82a6bafb8367ca4048e7688fb1d3d73efc5be49c829575440c7e7faad7f75f3201545fb3afed70acc56bf4f72c7cd8860b123e86fede91d8b28bda842cd99d9f55b669f18252b511123d672064370133983be8ebdd3f19636551c86bca9f64dc83d96f9c4e2c4bc427f3d8ad77e2e86460300642469522200f4219aa1a735047d566d0f2c063e147db5d0c9c272aa49050a86432372b14810a45cad16265e0efafc4252702239aa2f13d19303bbf095f545ef4f762593ab7bd101012169acf3233f3b32f321fb11d8e1f8cafe87acacf9b203b76299f0fd30bd08d828930e1c3d523d315ba1eef35a12da18e77e967cbd7418bce1c9bea1bb53a6323736257d46998099f68fd62daffc387e378b1dd8d238a09eeef814b88a8d32576c1a93900259118713a3724b8b38f66d2907f41dca75d90ec181de43455c1f91a72451ca500a9b10576f8a9d93ffaa4a342e06fbd0cea64e326dfd68f85c91708fd4f780c34ecda9f2f725e4ec8ff7aa3d9d565196b9409ad2efe6363fdca442ae21aaf333cd291f20235d7dad68883adf604d864588ba544c834921dae9e5d22ea73e25895107e5103db0c60041b566eff92a1f079751badf1b118f52cbc36c667ff18fa56d1a449b716c7f1ecf9bbdf12f34bb0a8d5bf2fa4ba3e55997550052d9ef906e243b38c7aa442aafcbd81ba75968d050cd164b1156c787700651531241898973ba463d1f281007312b82f0a09f23550933c8835465133f8772818b3469d2c58d37450229500b8e8aeab3b7d6703b9de59446131abcc91542ffe2d757849d5de8affc8afd9f48bdfb8c0fed84cb1204d529a6a87a71d436db07967735bfd86735f5cf5e1173ca5dcc3ef34f7eb5eaa4d39c6abd309c69d5eeca4893c603e44bfa9e5d345065903a462b3a68199aecea011609941886ffefdbb31d75fd43df5be4374a2ca83730645f1b2de60193138b7ce608afb960ec744995b141901a8296240485919593b5f3d4050de635c532befcfb5704e6a9e048caaeceaaf3ca034a35f9327f05919c8c75aee2bc959f4627266eca98e6682e13ef98ef29f708254c94bcedce45a71f2cf107a79958c787740e399fd9bffced95f863adc08782c1a4dc8af1f4e325906379d9b52c483386f203af23ecbbbe9c33831b6414e1db09f3b984ce939ba239751f4f8a455e372ceed1e58b4545f4d3c3edf5a9054f0de3927d554e23d8c98b1972ac66996ec7c8da1f3812f0cd55acb12f3817fa46951f122eb0cea51d5d6d5dccbc2c11b7f1fddc99469354659e76c64a1b4c3852254beb0b70a979763e48727e2926594af2f41bef9c7accd8ee4130e5312ff3140a883d1ec2d6d81bb35921a94afc499e55f725413b14cf98acccfccb0e4ff12f2ff6ea35f32d5079381aaedabd600dee6616be539d8d2c3acae60ff99312930547f74b11383ad68aef78da49d8badd36c8c2579beb3cfb5bfc54f3c33e245faccdfc8b1caf785f4d861052a5f35f50266e6c9bc3fb08fbc5c1e6666dd6d2fecf68cf31892b8a314d84a2b06ed9f7a0082b834cbf59b1996ac85b6b01e1dd1ff7d0b911c43901c6ea3a3dfd62796016a9a3eec46d4a9ef3f21c867726b5f257f3199b617db8508b02d23f04485ee7c9dab0ebf613e3a86f41c990c308a58a1e5e8f89ccf07fe34d1fd8dddeac9ed68cb2ff65aa43792a16137a827f9d499c9b03d66b04602a578f72c8169dd8b9a3b03d4f409a142549a6cd1c4388b0bdce1cb638a2a606d961291bef926d20cb6960d0c179bb6d2d0eaee46b88a7f3cd84c393a91852ccb78a70b602491fdee36087dde519bccd639e619224c607a7c1a80a781e3a3ba3900a
. . .
. . .
Invalid transaction, reason: Transaction fee is below the minimum required fee
Broadcasting transaction... !
✨  Done in 7.50s.
```

Reply:
Yes this code does not need to exist.